Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.cpp

```diff
@@ -32,6 +33,15 @@ namespace ROS2
                         &CameraSensorConfiguration::m_verticalFieldOfViewDeg,
                         "Vertical field of view",
                         "Camera's vertical (y axis) field of view in degrees.")
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
```

Comment:
What is a point of exposing both HFOV and VFOV for the User?
The horizontal field of view do not affect anything.
Please rectify it. I would recommend having an HFOV getter.

Reply:
Horizontal fov is now affecting the image. Please consider using this parameter as an increase in the flexibility of the camera module through a changeable viewing parameter.

Reply:
I understand that image resulution is now artifact of setting HFOV and VFOV?

Reply:
Changed to setting only vertical FOV